Libplanet/Address.cs

```diff
@@ -130,11 +131,11 @@ public ImmutableArray<byte> ByteArray
         public byte[] ToByteArray() => ByteArray.ToArray();
 
         /// <summary>
-        /// Gets a hexadecimal string of 40 letters that represent this
-        /// <see cref="Address"/>.
+        /// Gets a mixed-case hexadecimal string of 40 letters that represent
+        /// this <see cref="Address"/>.
         /// </summary>
```

Comment:
Could you append a mention that a returned hexadecimal string follows [EIP 55](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md)?

Reply:
I rebased on master and added a mention about EIP 55!